### ISSUE TITLE
why the connection is not work while i bridge to GNS3 VM nic?

### ISSUE THREAD

**Issue by Jamlee** [GNS3/gns3-server#1367]
I bridge the gns3 vm `eth0` to sw2 ` fa1/15`. but it is has problem to ping the `eth0 ip` from `sw2`. 

![image](https://user-images.githubusercontent.com/5196641/42418883-97c6d546-82dc-11e8-8106-59fe03292650.png)

With trying to slove the connection problem for three hours,   if  `adding  another nic whom has same network as eth0 ` to gns vm ,  i can ping the `eth0 ip` from sw2. like this:

![image](https://user-images.githubusercontent.com/5196641/42418889-bf3e721e-82dc-11e8-8b88-6e7dabb5e3d2.png)
![image](https://user-images.githubusercontent.com/5196641/42418895-ce7b5742-82dc-11e8-91d7-f74e996a97c9.png)

In summary,  **Does is it a BUG ? Why need two nics?**

**Comment by grossmj** [comment]
I recommend you ask the question on https://gns3.com/community, you will probably get a faster answer.
